app/scripts/views/components/scatterPlot/scatterPlot.js

```diff
@@ -165,9 +185,39 @@
     content.updateDataForDownload = function(fileType) {
       if (['pdf', 'svg'].indexOf(fileType) !== -1) {
         initCanvasDownloadData();
+      } else if (fileType === 'tsv') {
+        initTsvDownloadData();
       }
     };
 
+    function initTsvDownloadData() {
+      var _title = opts_.title || 'Mutation Count vs. CNA';
+      var _data = ['Patient ID', 'Sample ID', 'Mutation Count', 'CNA'];
+
+      if (groups_.length > 1) {
```

Comment:
I think about it. I was going to add download for survival plot, and survival has group concept for rainbow survival. Just for consistency, I changed to group in here as well. What do you think?